pkg/idp/provider/azure/azure.go

```diff
@@ -244,7 +247,8 @@ func (a *Azure) CreateOrUpdateSecret(id string, config provider.AppConfig, ctx c
 
 	// Create secret if it does not exist
 	if needsCreation {
-		secret, err = a.Client.ApplicationsById(id).AddPassword().Post(ctx, GetSecretCreateRequestBody(config), nil)
+		passwordRequestBody := applications.NewItemAddPasswordPostRequestBody()
```

Comment:
The old method incorporated some app config -- does that still get handled correctly?